cgi-bin/DW/Logic/ProfilePage.pm

```diff
@@ -851,6 +851,17 @@ sub external_services {
         };
     }
 
+    if ( my $medium = $u->prop( 'medium' ) ) {
+	my $medium = LJ::eurl( $medium );
+	push @ret, {
+	    type => 'medium',
+	    text => LJ::ehtml( $medium ),
+	    url => "https://medium.com/\@$medium",
```

Comment:
According to the spec in the issue text, I believe this URL should end with /latest?